### PR TITLE
JW-914 Compute-Controller integration tests.

### PR DESCRIPTION
**Pull request by m-kostrzewa**

### CODE REVIEW

File: CIScripts/Tests/TestConfigurationUtils.ps1 [hunk 1]
```diff
@@ -241,3 +245,72 @@ function Clear-TestConfiguration {
     Disable-VRouterExtension -Session $Session -AdapterName $TestConfiguration.AdapterName `
         -VMSwitchName $TestConfiguration.VMSwitchName -ForwardingExtensionName $TestConfiguration.ForwardingExtensionName
 }
+
+function New-AgentConfigFile {
```
Comment (m-kostrzewa) [inline CIScripts/Tests/TestConfigurationUtils.ps1:249]: FYI This func was moved from VRouterAgentTests.ps1

File: CIScripts/TestIntegration.ps1 [hunk 2]
```diff
@@ -37,11 +41,13 @@ $DockerDriverConfiguration = [DockerDriverConfiguration] @{
     Username = $Env:DOCKER_DRIVER_USERNAME;
     Password = $Env:DOCKER_DRIVER_PASSWORD;
     AuthUrl = $Env:DOCKER_DRIVER_AUTH_URL;
-    ControllerIP = $Env:DOCKER_DRIVER_CONTROLLER_IP;
     NetworkConfiguration = $DockerNetworkConfiguration;
 }
 
 $TestConfiguration = [TestConfiguration] @{
+    ControllerIP = $Env:CONTROLLER_IP;
```
Comment (m-kostrzewa) [inline CIScripts/TestIntegration.ps1:48]: You'll notice that I moved this field from `DockerDriverConfiguration` struct and renamed the field. Already added new fields to jenkins job params. I kept `DOCKER_DRIVER_CONTROLLER_IP` to not break anything but we'll need to remove it eventually.

File: CIScripts/Tests/ComputeControllerIntegrationTests.ps1 [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+    }
+
+    function Get-PSHostname {
+        Param([Parameter(Mandatory = $true)] [PSSessionT] $Session)
+        return Invoke-Command -Session $Session -ScriptBlock { hostname }
+    }
+
+    function Get-PSIPAddress {
+        Param([Parameter(Mandatory = $true)] [PSSessionT] $Session)
+        return Invoke-Command -Session $Session -ScriptBlock { Get-NetIPAddress | Where-Object InterfaceAlias -like "Ethernet0*" | Where-Object AddressFamily -eq IPv4 | Select-Object -ExpandProperty IPAddress }
```
Comment (kaweue) [inline CIScripts/Tests/ComputeControllerIntegrationTests.ps1]: Zastanawiam się czy ta linia nie jest trochę za długa.

**Review by sodar — DISMISSED** [review]

**Review by kaweue — DISMISSED** [review]

**Comment by m-kostrzewa** [comment]
Decided to remove cassandra test from this PR, since I need to rethink it

**Review by kaweue — APPROVED** [review]

**Review by sodar — APPROVED** [review]

**Pull request merged**